app/femr/ui/models/admin/configure/IndexViewModelGet.java

```diff
@@ -4,21 +4,33 @@
 import java.util.Map;
 
 public class IndexViewModelGet {
-    private Map<String, Boolean> settings;
+    private Map<String, Boolean> settings;;
+    private Map<String, String> descriptions;;
 
```

Comment:
@judyvbaxter can you comment what the Maps contain so someone new can understand why we have 2 maps here? :+1: